src/EditorFeatures/Core/Shared/Extensions/TelemetryExtensions.cs

```diff
@@ -28,15 +28,13 @@ public static Type GetTypeForTelemetry(this Type type)
         }
 
         public static short GetScopeIdForTelemetry(this FixAllScope scope)
-        {
-            switch (scope)
+            => (short)(scope switch
```

Comment:
We have target typed switch expressions now, right? Maybe we want to wait until it gets into a stable release of VS before adopting?

Reply:
i'm ok with this as is.  at some point we'll then get the suggestion that this cast can be removed.